src/operator/image/normalize_op-inl.h

```diff
@@ -0,0 +1,181 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * Copyright (c) 2018 by Contributors
+ * \file normalize_op-inl.h
+ * \brief Image normalization operator
+*/
+#ifndef MXNET_OPERATOR_IMAGE_NORMALIZE_OP_INL_H_
+#define MXNET_OPERATOR_IMAGE_NORMALIZE_OP_INL_H_
+
+
+#include <mxnet/base.h>
+#include <algorithm>
+#include <vector>
+#include <cmath>
+#include <limits>
+#include <algorithm>
+#include <utility>
+#include "../mxnet_op.h"
+#include "../operator_common.h"
+#include "../elemwise_op_common.h"
+
+namespace mxnet {
+namespace op {
+namespace image {
+
+// Parameter registration for image Normalize operator
+struct NormalizeParam : public dmlc::Parameter<NormalizeParam> {
+  nnvm::Tuple<float> mean;
+  nnvm::Tuple<float> std;
+  DMLC_DECLARE_PARAMETER(NormalizeParam) {
+    DMLC_DECLARE_FIELD(mean)
```

Comment:
mean and std can have default value (0, and 1.0) for conveniency

Reply:
Done.